packages/lesswrong/components/admin/CurationPage.tsx

```diff
@@ -0,0 +1,101 @@
+// TODO: Import component in components.ts
+import React, { useState } from 'react';
+import { Components, getFragment, registerComponent } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import { useCurrentUser } from '../common/withUser';
+import { useMulti } from '../../lib/crud/withMulti';
+import { userCanDo } from '@/lib/vulcan-users';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+
+  },
+  curated: {
+    position: "absolute",
+    right: 0,
+    top: 65,
+    width: 210,
+    [theme.breakpoints.down('md')]: {
+      display: "none"
+    }
+  }
+});
+
+export const CurationPage = ({classes}: {
+  classes: ClassesType<typeof styles>,
+}) => {
+  const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
+  const currentUser = useCurrentUser()
+
+  const { SunshineCuratedSuggestionsList, SingleColumnSection, BasicFormStyles, WrappedSmartForm, SectionTitle, ErrorAccessDenied, CurationNoticesItem, CommentsList } = Components
+
+  const [ post, setPost ] = useState<PostsList|null>(null)
+
+  const { results: curationNotices = [], loading } = useMulti({
+    collectionName: 'CurationNotices',
+    fragmentName: 'CurationNoticesFragment',
+    terms: {
+      view: "curationNoticesPage",
+      limit: 20
+    }
+  });
+  console.log(curationNotices.map((notice) => ({title: notice.post?.title, commentId: notice.commentId})))
+
+  const curationNoticesList = curationNotices.filter(notice => !notice.comment);
+  const curationCommentsList = curationNotices.filter(notice => notice.comment).map(notice => notice.comment!);
+
+  const commentTreeNodes = curationCommentsList.map(comment => ({
```

Comment:
I believe you want `unflattenComments` here, rather than manually mapping over the comments

Reply:
thx! it now reads

>  const commentTreeNodes = unflattenComments(curationCommentsList);